app/assets/javascripts/soundcloud_stream.js.erb

```diff
@@ -1,50 +1,42 @@
-$(document).ready(function(){
-  SC.initialize({
-    client_id: "<%= ENV['SOUNDCLOUD_APP_ID'] %>",
-    redirect_uri: 'http://cryptic-dusk-7305.herokuapp.com/callback'
-  });
-
-  $('#connect').click(function(){
-    var currentUser = new SoundcloudUser
-    currentUser.searchSongs();
-  })
+$('#connect').click(function(){
+  new SoundcloudUser
+})
 
-  $('.track').on('click', function(event){
-    track = new Track({id: event.currentTarget.dataset.trackId, source_id: event.currentTarget.dataset.sourceId})
-    track.audio.play()
-  })
+$('.user-tracks').on('click', '.track', function(event){
+  track = new Track({id: event.currentTarget.dataset.trackId, source_id: event.currentTarget.dataset.sourceId})
+  track.audio.play()
+})
 
-  $('.snippet').on('click', function(event){
-    var snipData = event.currentTarget.dataset;
-    var snippetTrack = new Track({source_id: snipData.sourceId});
-    snippet = new Snippet({track: snippetTrack, startTime: snipData.startTime, endTime: snipData.endTime
-    });
-    snippet.playSnip();
-  })
+$('.user-snippets').on('click', '.snippet', function(event){
+  var snipData = event.currentTarget.dataset;
+  var snippetTrack = new Track({source_id: snipData.sourceId});
+  snippet = new Snippet({track: snippetTrack, startTime: snipData.startTime, endTime: snipData.endTime
+  });
+  snippet.playSnip();
+})
 
-  $('body').on('click', '#pause', function(event){
-    track.audio.pause()
-    $(this).replaceWith("<div id='play'>PLAY~</div>")
-  })
+$('.columns#right').on('click', '#pause', function(event){
+  track.audio.pause()
+  $(this).replaceWith("<div id='play'>PLAY~</div>")
+})
 
-  $('body').on('click', '#play', function(event){
-    track.audio.play()
-    $(this).replaceWith("<div id='pause'>PAUSE~</div>")
-  })
+$('.columns#right').on('click', '#play', function(event){
+  track.audio.play()
+  $(this).replaceWith("<div id='pause'>PAUSE~</div>")
+})
 
-  $('body').on('click', '#snip', function(event){
-    var endTime = track.audio.currentTime;
-    var startTime = track.audio.currentTime - 15;
-    var newSnippet = new Snippet(
-      {track: track,
-      startTime: startTime,
-      endTime: endTime});
-    $.ajax({
-      url: '/snippets',
-      method: 'POST',
-      data: {snippet: {track_id: newSnippet.track_id, start_time: newSnippet.start_time, end_time: newSnippet.end_time}},
-    }).done(function(response){
-      $('.user-snippets').append(response);
-    })
+$('.columns#right').on('click', '#snip', function(event){
+  var endTime = track.audio.currentTime;
+  var startTime = track.audio.currentTime - 15;
```

Comment:
`15` is a "MAGIC NUMBER".  Why 15?  Why not 17.5 or 3.14?  A classic refactoring is

"REPLACE MAGIC NUMBER WITH SYMBOLIC CONSTANT"

So like

`DEFAULT_SAMPLE_SIZE = 15` 

and then 

` track.audio.currentTime - DEFAULT_SAMPLE_SIZE`

that tells me much more.  think 'bout it.